src/install/extract_tarball.zig

```diff
@@ -309,7 +309,7 @@ fn extract(this: *const ExtractTarball, tgz_bytes: []const u8) !Install.ExtractD
     // e.g. @next
     // if it's a namespace package, we need to make sure the @name folder exists
     if (basename.len != name.len and !this.resolution.tag.isGit()) {
-        cache_dir.makeDir(std.mem.trim(u8, name[0 .. name.len - basename.len], "/")) catch {};
+        cache_dir.makePath(std.mem.trim(u8, name[0 .. name.len - basename.len], "/")) catch {};
```

Comment:
is the trailign slash correct

should it be std.fs.path.sep

Reply:
yes because its from a normalized package name like `@foo/bar`
trimming basename gives `@foo/`
then trimming / gives `@foo`